src/main.cpp

```diff
@@ -1195,6 +1267,7 @@ void loopPid() {
 
     checkSteamSwitch();
     checkPowerSwitch();
+    checkWaterSwitch();
```

Comment:
checkWaterSwitch();

i would move that into bool hotWater()

Reply:
I think it's fine this way. The other "check" methods also have an early return. Keeps main.cpp a bit cleaner.